arm/Microsoft.Sql/servers/deploy.bicep

```
@@ -43,6 +43,9 @@ param firewallRules array = []
 @description('Optional. The security alert policies to create in the server')
 param securityAlertPolicies array = []
 
+@description('Conditional. The Azure Active Directory (AAD) administrator authentiaction.')
```

We currently don't support 'Conditional' as a keyword (Pester will aks for `Required`, `Optional` or `Genererated`). However, might be worth adding in a subsequent PR.

```suggestion
@description('Optional. The Azure Active Directory (AAD) administrator authentiaction.')
```